rust-toolchain.toml

```
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "1.55"
+channel = "1.56"
```

```suggestion
channel = "1.56.1"
```

I believe that that's the current stable

---

It will auto-pick the highest patch release AFAIK.